### PR TITLE
Update zha.markdown with Zigbee device limits information

### PR DESCRIPTION
**Pull request by Hedda**
## Proposed change

Update zha.markdown with Zigbee coordinator and Zigbee routers device limits information based on Zigbee2MQTT FAQ https://www.zigbee2mqtt.io/information/FAQ.html#i-read-that-zigbee2mqtt-has-a-limit-of-20-devices-is-this-true

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: NA
- Link to parent pull request in the Brands repository: NA
- This PR fixes or closes issue: NA

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/zha.markdown [hunk 1]
```diff
@@ -224,6 +224,19 @@ To add a new device:
...
 1. Once the device is found, it will appear on that page and will be automatically added to your devices. You can optionally change its name and add it to an area (you can change this later). You can search again to add another device, or you can go back to the list of added devices.
 
+### Using router devices
+
+You use routers to increase the number of of Zigbee devices that can be used in a network. The total number of Zigbee devices that you have on a Zigbee network depends on a few things, nut you should know that Zigbee coordinator hardware and firmware only plays larger roll in Zigbee networks with a lot of devices. More important is how many directly connected devices ("direct childred") verses how many routers are connected to your Zigbee coordinator.  Zigpy library which ZHA uses has a upper limit is 32 direct children but if your Zigbee coordinator hardware is powerful enough then you can still have hundreds of Zigbee devices connected through routers.
```
Comment (frenck) [inline source/_integrations/zha.markdown]:
```suggestion
You use routers to increase the number of Zigbee devices that can be used in a network. The total number of Zigbee devices that you have on a Zigbee network depends on a few things, but you should know that Zigbee coordinator hardware and firmware only plays a larger role in Zigbee networks with a lot of devices. More important is how many directly connected devices ("direct children") versus how many routers are connected to your Zigbee coordinator. Zigpy library which ZHA uses has an upper limit is 32 direct children but if your Zigbee coordinator hardware is powerful enough then you can still have hundreds of Zigbee devices connected through routers.  
```

File: source/_integrations/zha.markdown [hunk 2]
```diff
@@ -224,6 +224,19 @@ To add a new device:
...
+
+You use routers to increase the number of of Zigbee devices that can be used in a network. The total number of Zigbee devices that you have on a Zigbee network depends on a few things, nut you should know that Zigbee coordinator hardware and firmware only plays larger roll in Zigbee networks with a lot of devices. More important is how many directly connected devices ("direct childred") verses how many routers are connected to your Zigbee coordinator.  Zigpy library which ZHA uses has a upper limit is 32 direct children but if your Zigbee coordinator hardware is powerful enough then you can still have hundreds of Zigbee devices connected through routers.  
+
+Even the lest powerful Zigbee coordinator hardware supported by Zigpy is CC2530/2531 and its default firmware only support 20 devices connected directly to the coordinator. However, by having routers in your Zigbee network the mesh network size can be extended. You can assume that most if not all mains/AC-powered devices e.g. wall-plugs and always powered-on lightbulbs in your Zigbee network can serve as a router, you can even use CC2530/CC2531 with router firmware as your routers (which in turn has a limit of 21 devices).
```
Comment (frenck) [inline source/_integrations/zha.markdown]:
```suggestion
Even the least powerful Zigbee coordinator hardware supported by Zigpy is CC2530/2531 and its default firmware, only supports 20 devices connected directly to the coordinator. However, by having routers in your Zigbee network the mesh network size can be extended. You can assume that most, if not all mains/AC-powered devices, e.g., wall-plugs and always powered-on lightbulbs in your Zigbee network, can serve as a router. You can even use CC2530/CC2531 with router firmware, as additional routers (which in their turn have a limit of 21 devices).
```

File: source/_integrations/zha.markdown [hunk 3]
```diff
@@ -224,6 +224,19 @@ To add a new device:
...
+
+An example is using the defaultCC2531 coordinator firmware + two CC2531 routers your device limit will be:
```
Comment (frenck) [inline source/_integrations/zha.markdown]:
```suggestion
An example using the default CC2531 coordinator firmware + two CC2531 routers; Your device limit will be:
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by Hedda** [comment]
Thanks for the review!

**Review by frenck — APPROVED** [review]
Thanks! 👍

**Pull request merged**